knowledge-base/kb-ribbonview-override-esc-key-logic-in-backstage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Override Escape Key Logic of RadRibbonBackstage
+page_title: Change RadRibbonBackstage Logic for Escape Key
```

Comment:
Use "Esc" instead of "Escape" in the metadata (in title, page_title or description).